src/main/kotlin/fi/metatavu/timebank/api/severa/models/SeveraUser.kt

```diff
@@ -0,0 +1,24 @@
+package fi.metatavu.timebank.api.severa.models
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.quarkus.runtime.annotations.RegisterForReflection
+
+/**
+ * Data class for User data coming from Severa
+ */
+@RegisterForReflection
+@JsonIgnoreProperties(ignoreUnknown = true)
+class SeveraUser {
+    var guid: String = ""
+    @JsonProperty("firstName")
+    var firstName: String = ""
+    @JsonProperty("lastName")
+    var lastName: String = ""
+    var email: String = ""
+    @JsonProperty("startDate")
+    var startDate: String = ""
+    @JsonProperty("endDate")
+    var endDate: String? = null
```

Comment:
`startDate` and `endDate` fields seem to be included in a sub-object of Severas `UserOutputModel` called `UserWorkContractSubModel`. You should lookup Jacksons documentation on how to handle sub-objects.